cvat/apps/dataset_manager/util.py

```diff
@@ -111,14 +111,17 @@ def get_export_cache_lock(
     *,
     ttl: int | timedelta,
     block: bool = True,
-    acquire_timeout: Optional[int | timedelta] = None,
+    acquire_timeout: int | timedelta,
 ) -> Generator[Lock, Any, Any]:
+    if acquire_timeout is None:
+        raise ValueError("Endless waiting for the lock should be avoided")
+
```

Comment:
`None` is not expected, according to the argument annotations. Maybe the check should be removed.

Reply:
> None is not expected, according to the argument annotations. 

But it was expected in the previous implementation+ I think we must prevent endless waiting for lock acquisition. Updated.